ed_sensor_integration/include/ed/kinect/fitter.h

```diff
@@ -103,17 +105,22 @@ class Fitter
     /**
      * @brief Fitter constructor
      * @param nr_data_points nr_data_points for the beam model
+     * @param fx focal length for the beam model
      */
-    Fitter(uint nr_data_points = 200);  // TODO: remove hard-coded values
+    Fitter();
```

Comment:
I think we should get rid of the default constructor. All constructor should have their own docstring.